### PR TITLE
Update community page

### PR DESCRIPTION
**Pull request by eliasmbd**
Here is the updated community layout. 

Still need to review links

### CODE REVIEW

**Comment by eliasmbd** [comment]
Replaces #15

File: docs/community/get_involved.md
Comment (Sergiodero) [inline docs/community/get_involved.md]:
```suggestion
To formalize this process, we've established the GTFS Governance framework, which includes the Specification Amendment Process.
```

File: docs/community/get_involved.md
Comment (Sergiodero) [inline docs/community/get_involved.md:22]: We need to update this link once PR #30 is merged, you can ignore this comment when merging, it's just a reminder

File: docs/community/get_involved.md
Comment (Sergiodero) [inline docs/community/get_involved.md:26]: Same as avobe: update this link once PR #30 is merged

File: docs/community/get_involved.md
Comment (Sergiodero) [inline docs/community/get_involved.md:14]: This text is included in each Spec Amendment Process, should we remove it from there and keep it in here?

**Comment by Sergiodero** [comment]
Thanks for working on this! The page is displaying correctly and it's looking good, I only left a [suggestion](https://github.com/MobilityData/elrond.gtfs.org/pull/27#discussion_r1603991786) to cut the SAP acronym. 

We still need to review all links, so we can either merge as is and open a new PR that only focuses on this, or wait for other PRs to be merge since this is dependant on them (PR #30 and PR #29).

I'm okay with both,  so you can decide. LMK and I can approve afterwards. Thanks!

File: docs/community/get_involved.md
Comment (eliasmbd) [reply docs/community/get_involved.md:14]: id keep it

**Comment by eliasmbd** [comment]
> Thanks for working on this! The page is displaying correctly and it's looking good, I only left a [suggestion](https://github.com/MobilityData/elrond.gtfs.org/pull/27#discussion_r1603991786) to cut the SAP acronym.
> 
> We still need to review all links, so we can either merge as is and open a new PR that only focuses on this, or wait for other PRs to be merge since this is dependant on them (PR #30 and PR #29).
> 
> I'm okay with both, so you can decide. LMK and I can approve afterwards. Thanks!

Let's merge this one and then create other PR for the subsequent steps.

**Review by Sergiodero — APPROVED** [review]
Perfect, I'm approving! Thanks again!

**Pull request merged**
